### ISSUE TITLE
ableist results detail box is missing ARIA label

### ISSUE THREAD

**Issue by dianalam** [USDepartmentofLabor/ableist-language-detector-web#2]
The box that shows alternative language does not appear to have an ARIA label, and therefore will not be keyboard accessible.
